content/blog/technical-assessment-evaluation-criteria.md

```diff
@@ -0,0 +1,104 @@
+
+## Technical Discussion with our existing team members
+A 30-minute discussion around your technical background and journey so far. You can expect the conversation to revolve around languages, frameworks, and databases you have worked with; your prior experiences, projects, etc. This is for us to get introduced to the depth and breadth of your technical prowess. Here it is important that you have a good Wi-Fi connection since we'd want to have your video on during the discussion.
+
+## Software Craft Round
+Congratulations! You have gotten to this round. Here is where all the fun happens. You will fire up your IDE with the language of your choice and a simple Hello World program pre-installed and get ready to get your hands dirty. We will solve a short Kata together as a team! Consider this round as a pairing exercise, in which an Incubyte developer will partner with you and act as a navigator. It is fine to look stuff up on the internet. We understand that having Maven or Gradle dependency coordinate on top of the code is difficult. (We google things while working on production projects ALL the time!)
+
+This brings us to the end of our interview process, expect to walk away with an offer once you have cleared all these rounds. 
+
+Now comes the important part, how to clear this interview round? We have broken down our requirements into 2 parts:
+1. Technical
+2. Non-technical  
+
+## Technical
+We'd love to see how you code and how you think. What matters to you and how does it align with what matters to us. Here are some of the technical things that matter most to us:
+  
+### Build tools like Maven or Gradle
+
+We believe that build tools are first-class members of our development life cycle. Choosing the best build and dependency management tool when beginning a new project is vital and can go a long way.
+
+### Clean coding practices
+
+It is not possible to evaluate everything during an interview, but below mentioned are a few key indicators that give us an insight into how you work:
+
+#### Code format
+
+Code writing skills are essential, but so is code readability. A neatly formatted code is readable.
+
+#### Amount of nesting in your code
+
+Excessive nesting makes the code difficult to understand and raises cognitive complexity.
+
+#### Applying best practices to your code:
+
+Using clean code practices speaks much more of a personal habit as much as your coding skill. It can range from using the appropriate naming conventions to Page Object Models if you are working on test automation. Having some linters integrated into your IDE can help you adhere to best practices.
+
+#### Commented-out code
```

Comment:
Too detailed and part of clean code already. I would consider removing all these details from this section. Also, headings are not consistent in sizes

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/1727880/174306207-db773e01-3eca-4181-b371-59695d9e1d63.png">